DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,7 +56,19 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
+        // TODO: To be removed when major version is changed
```

Comment:
After we merged this PR, please create an issue and set the next major version as milestone, that we don't forget this!

Thank you @sagikazarmark